.gitignore

```diff
@@ -9,6 +9,9 @@ _test-output
 cypress.zip
 .babel-cache
 
+# from config, compiled .js files
+packages/config/src/*.js
```

Comment:
Would it be better to have a dist folder instead?

How did those js files get created in `@packages/config`? I do not see a tsc script.

Reply:
You added this: https://github.com/cypress-io/cypress/pull/20590/files#diff-fd956f6d78d80e1989716c00afb66f1ac64847e9b4a3673def6f483cd218e963R9

Reply:
Then you have the right call. 
I don't run build-prod on my local often. I should have.